components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -626,6 +523,135 @@ private void readChildElements(OMElement serverConfig,
         }
     }
 
+    /**
+     * Set property values for each gateway environments defined in the api-manager.xml config file
+     *
+     * @param environmentElem OMElement of a single environment in the gateway environments list
+     */
+    void setEnvironmentConfig(OMElement environmentElem) throws APIManagementException {
+
```

Comment:
remove new line.check in other places as well

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10839/commits/20106d3b631c172110c54b848257432bf17bb19d